doc/api/net.md

```diff
@@ -331,10 +331,18 @@ Returns `server`.
 added: v0.3.4
 -->
 
-This object is an abstraction of a TCP or local socket.  `net.Socket`
-instances implement a duplex Stream interface.  They can be created by the
-user and used as a client (with [`connect()`][]) or they can be created by Node.js
-and passed to the user through the `'connection'` event of a server.
+This class is an abstraction of a TCP socket, a UNIX domain socket, or
+a Windows named pipe. A `net.Socket` is also a [duplex stream][], so it
```

Comment:
I have considered using the term IPC but remembered people are using `net` abstractions over Windows named pipes for RPC as well? (I think I've seem issues about access permissions before).

So maybe: `a abstraction of a TCP or streaming socket (uses named pipes on Windows, and UNIX domain sockets otherwise)` (loses the local part)?